README.rst

```diff
@@ -150,6 +150,24 @@ keys:
     surrounding text, and so lookahead and lookbehind will not work in this
     regex.
 
+``datalad``
+    *(optional)* A sub-mapping describing integration of ``tinuous`` with
+    Datalad_.  Subfields:
+
+    ``enabled``
+        *(optional)* A boolean.  If true (default false), the current directory
+        will be converted into a Datalad dataset if it is not one already,
+        the logs will be divided up into subdatasets, and all new logs will be
+        committed at the end of a run of ``tinuous fetch``.  All ``path``
+        template strings must contain a ``//`` separator; the part to the left
+        of the separator will be used as the path to the subdataset.
```

Comment:
Some day I will learn to be exhaustive in the description of my design wishes!
- I think `//` should not be made mandatory: for some "slow" projects with small logs it would be totally fine to have no subdatasets. 
- What I really wished is to allow for multiple `//` to separate possibly subdatasets per each year and then month.  But I think pragmatically it is not really needed (yet at least).  So it is ok to allow for up to a single `//` but let's make it explicit - i.e blow up with `NotImplementedError("Multiple subdataset boundaries are not supported")` if multiple `//` found in the expanded path.  I will leave it up to you either to support an arbitrary number of subdatasets or up to 1 (+exception)

Reply:
`//` separators are now optional and can occur multiple times.